Recentupdate

```diff
@@ -0,0 +1,266 @@
+<?php
+$strValidateMessage = "";
+
+
+
+// check for form submission
+if (isset($_POST['submit'])) {
+
+    // inputs from browser
+	$name     = trim($_POST['name']);
+	$phone    = trim($_POST['phone']);
+	$email    = trim($_POST['email']);
+	$suburb   = trim($_POST['suburb']);
+	$dropdown = trim($_POST['dropdown']);
+	$message  = trim($_POST['message']);
+
+
+    // email recipient as subject
+	$to      = "contact@rawadmerhi.com";
+	$subject = "New Message";
+
+    // validate inputs
+	$errors = validateInputs($name,$phone,$email,$suburb,$message);
+
+	if (empty($errors)) {
+         // no errors, send the mail
+		if (mail($to, $subject, $message, "From: $name <$email>")) {
```

Comment:
^ Fix my wacky indention up there. But after we add those three lines, we can change the third parameter of `mail()` from `$message` to `$email_body`